strings/tcg/set_one.json

```diff
@@ -1591,23 +1591,23 @@
 			"resolve": "0",
 			"faction": "Medical",
 			"summoncost": "3",
-			"cardtype": "Event",
-			"cardsubtype": "Instant",
+			"cardtype": "Instant",
+			"cardsubtype": "Clown Medical",
 			"series": "coreset2020",
 			"rarity": "uncommon"
 		},
 		{
 			"id": "disco_inferno",
 			"name": "Disco Inferno",
 			"desc": "(Burn baby burn) burn that mother down y'all\n(Burn baby burn) Disco Inferno\n(Burn baby burn) burn that mother down",
-			"rules": "For 2 mana: Tap this card permanantly. While tapped, all active creatures take 3 damage during the first play phase of each turn. This card is destroyed after 2 turns of being tapped.",
+			"rules": "All active creatures take 3 damage during the first play phase of each turn. This card is destroyed after 4 play phases.",
```

Comment:
For clarity, the original text specifies that 4 play phases must conclude after the effect begins (ala: swords of revealing light), otherwise this could be interpreted as being destroyed being on the 4th turn of the game if played earlier than that.

Reply:
Changed to "This card is destroyed after this effect triggers 4 times."